DfE.FindInformationAcademiesTrusts/Program.cs

```diff
@@ -57,6 +57,7 @@ public static void Main(string[] args)
 
     private static void ConfigureHttpRequestPipeline(WebApplication app)
     {
+        app.UseExceptionHandler("/ProblemWithProduct");
```

Comment:
This should be inside the if on line 63 (I think we should also change the if to remove `!app.Environment.IsDevelopment()`)